lib/send-rec/sender.js

```diff
@@ -40,12 +44,13 @@ Sender.prototype.produce = function ( message, callback ) {
 	}
 
 	this.send( message, function () {
+		this.logger.info( 'Message sent' );
```

Comment:
I don't think we're going to want to log every time a message is sent. By default I think that we're going to have our levels set at info. I think that we may want to put this at `debug` or `silly`.